specs/eventing/data-plane.md

```diff
@@ -1,125 +1,171 @@
-sent again. If the event can not be delivered then some sources of events (such
-as Knative sources, brokers or channels) MAY support
-[dead letter sink or channel](https://github.com/knative/eventing/blob/main/docs/delivery/README.md) for events that can not be
-delivered.
+All senders and recipients MUST support the CloudEvents 1.0 protocol and the
+[binary](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#31-binary-content-mode)
+and
+[structured](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#32-structured-content-mode)
+content modes of the CloudEvents HTTP binding. Senders which do not advertise
+the ability t o accept [reply events](#derived-reply-events) MAY implement only
+one content mode, as the recipient is not allowed to negotiate the content mode.
+Senders MUST support both cleartext (`http`) and TLS (`https`) URLs as event
+delivery destinations.
+
+### HTTP Verbs
+
+In the absence of specific delivery preferences, the sender MUST initiate
+delivery of the event to the recipient using the HTTP POST verb, using either
+the structured or binary encoding of the event (sender's choice). This delivery
+SHOULD be performed using the CloudEvents HTTP Binding, version 1.0.
+
+Senders MAY probe the recipient with an
+[HTTP OPTIONS request](https://tools.ietf.org/html/rfc7231#section-4.3.7); if
+implemented, the recipient MUST indicate support for the POST verb using the
+[`Allow` header](https://tools.ietf.org/html/rfc7231#section-7.4.1). Senders
+which receive an error when probing with HTTP OPTIONS SHOULD proceed using the
+HTTP POST mechanism.
+
+### Event Acknowledgement and Repeat Delivery
+
+Event recipients MUST use the HTTP response code to indicate acceptance of an
+event. The recipient SHOULD NOT return a response accepting the event until it
+has handled event (processed the event or stored it in stable storage). The
+following response codes are explicitly defined; event recipients MAY also
+respond with other response codes. A response code not in this table SHOULD be
+treated as a retriable error.
+
+| Response code | Meaning                     | Retry | Delivery completed | Error |
+| ------------- | --------------------------- | ----- | ------------------ | ----- |
+| `1xx`         | (Unspecified)               | Yes\* | No\*               | Yes\* |
+| `200`         | [Event reply](#event-reply) | No    | Yes                | No    |
+| `202`         | Event accepted              | No    | Yes                | No    |
+| other `2xx`   | (Unspecified)               | Yes\* | No\*               | Yes\* |
+| other `3xx`   | (Unspecified)               | Yes\* | No\*               | Yes\* |
+| `400`         | Unparsable event            | No    | No                 | Yes   |
+| `404`         | Endpoint does not exist     | Yes   | No                 | Yes   |
+| other `4xx`   | Error                       | Yes   | No                 | Yes   |
```

Comment:
see here for `409` and `429` as well: https://github.com/knative/eventing/blob/main/pkg/kncloudevents/message_sender.go#L192-L204

Reply:
I switched this and called out `404`, `409`, and `429` explicitly.